account_payment_order/tests/test_payment_order_outbound.py

```diff
@@ -407,6 +407,7 @@ def test_supplier_refund_reference(self):
         self.refund = self._create_supplier_refund(self.invoice)
         with Form(self.refund) as refund_form:
             refund_form.ref = "R1234"
+            refund_form.payment_reference = "FR/1234"
```

Comment:
This is not tested anywhere

Reply:
This data is set and then checked to make sure that this value has not been set in communication field.